components/analytics/doughnut_chart.test.tsx

```diff
@@ -74,10 +75,11 @@ describe('components/analytics/doughnut_chart.jsx', () => {
                 width={600}
                 data={data}
             />
-        );
+        ) as ReactWrapper<{}, {}, DoughnutChart>;
 
         expect(Chart).toBeCalled();
-        const chartDestroy = wrapper.instance().chart.destroy;
+        const chartObject = wrapper.instance().chart;
```

Comment:
Instead of having to do a null check on a separate line, you should be able to use the `!.` operator to do an implicit null check.
```suggestion
        const chartObject = wrapper.instance().chart!.destroy;
```

Reply:
Your lint configuration does not allow that, unless you changed it.

Reply:
Sorry about that! I've fixed the lint configuration, if you merge in master you should be able to go ahead and use the `!.` operator.

Reply:
Fixed.